django_freeradius/api/serializers.py

```diff
@@ -63,3 +65,53 @@ def validate(self, data):
     class Meta:
         model = RadiusAccounting
         fields = '__all__'
+
+
+class GroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Group
+        fields = '__all__'
+
+
+class UserSerializer(serializers.ModelSerializer):
+    groups = GroupSerializer(many=True)
+
+    class Meta:
+        model = User
+        fields = '__all__'
+
+
+class RadiusBatchCsvSerializer(serializers.ModelSerializer):
+    name = serializers.CharField()
+    strategy = serializers.CharField()
+    users = UserSerializer(many=True, read_only=True)
+    csvfile = serializers.FileField()
+    prefix = serializers.CharField(required=False, read_only=True)
+    expiration_date = serializers.DateField(required=False)
+    pdf = serializers.FileField(required=False, read_only=True)
+
+    class Meta:
+        model = RadiusBatch
+        fields = '__all__'
+
+    def create(self, validated_data):
+        return RadiusBatch.objects.create(**validated_data)
+
+
+class RadiusBatchPrefixSerializer(serializers.ModelSerializer):
```

Comment:
there's a lot of duplication with the other serializer, can you define the common fields in a shared class?